packages/@apphosting/adapter-nextjs/src/bin/build.ts

```diff
@@ -2,12 +2,58 @@
 import { spawnSync } from "child_process";
 import { loadConfig, readRoutesManifest } from "../utils.js";
 
+import { join } from "path";
+import fsExtra from "fs-extra";
+import { stringify as yamlStringify } from "yaml";
+
+const { move, exists, writeFile, mkdirp } = fsExtra;
 const cwd = process.cwd();
 
-spawnSync("next", ["build"], { cwd, stdio: "inherit", env: { ...process.env, NEXT_PRIVATE_STANDALONE: 'true' } });
+// Set standalone mode
+process.env.NEXT_PRIVATE_STANDALONE = "true"; 
+// Opt-out sending telemetry to Vercel
+process.env.NEXT_TELEMETRY_DISABLED = "1"; 
+
+build(cwd);
+
+const {distDir, basePath} = await loadConfig(cwd);
+const manifest = await readRoutesManifest(join(cwd, distDir));
+
+const destination = join(cwd, ".apphosting");
+const staticDestination = join(destination,"_next", "static");
+const publicDestination = join(destination, "public");
+const outputBundleDestination = join(destination, "bundle.yaml");
+
+const standaloneDir = join (cwd, distDir, "standalone");
+const staticDir = join (cwd, distDir, "static");
+const publicDir = join(cwd, "public");
+
+await mkdirp(staticDestination);
+
+// Run build command
+function build(cwd: string) {
+    spawnSync("npm run", ["build"], {cwd, stdio: "inherit"}); 
+  }
 
-const config = await loadConfig(cwd);
-const routeManifest = await readRoutesManifest(config.distDir);
+// move public directory to both public and CDN destination
```

Comment:
This code below just moves the public directory to the public directory under .apphosting right? I don't think CDN destination is involved.

Also for now don't worry about moving anything to a CDN directory I think this logic is good already

Reply:
good catch! forgot to remove that when I remove the CDN directory